hw03_frequency_analysis/go.mod

```
@@ -1,10 +1,9 @@
-module github.com/fixme_my_friend/hw03_frequency_analysis
+module github.com/IgorSemenov70/hw-golang-otus/tree/master/hw03_frequency_analysis
```

```suggestion
module github.com/IgorSemenov70/hw-golang-otus/hw03_frequency_analysis
```
`tree/master` - часть роутинга в UI GitHub, не относится к модулям в Go